release-calendar/src/client/components/App.tsx

```diff
@@ -15,39 +15,106 @@
  */
 import * as React from 'react';
 import {Calendar} from './Calendar';
+import {Cell, Grid, Row} from '@material/react-layout-grid';
+import {ChannelTable} from './ChannelTable';
 //TODO: remove DATA and instead populate with data from ./test/development-data.ts
+import '@material/react-card/index.scss';
+import '@material/react-layout-grid/index.scss';
 import {DATA} from '../models/data';
 import {EventSourceInput} from '@fullcalendar/core/structs/event-source';
 import {Header} from './Header';
+import {getChannelRTVs} from '../models/release-channel';
```

Comment:
Does this have to be called at the `App` level, or can it be moved to the `ChannelTable component`?

Reply:
that's a good question. If i were to move it to the `ChannelTable component` I would probably need to sent all the raw DATA as a parameter to the component. do we like that or should I filter the data in some other way beforehand?

Reply:
You can create an API call specific to `ChannelTable` that gets current releases for all channels. This would be different from DATA which I assume gets releases by month.

Reply:
DATA actually wont. FullCalendar adapts under the hood to changes in month. I can send it all the release events and it will organize the display for me.